test/parallel/test-crypto-sign-verify.js

```diff
@@ -15,6 +15,42 @@ const certPem = fixtures.readSync('test_cert.pem', 'ascii');
 const keyPem = fixtures.readSync('test_key.pem', 'ascii');
 const modSize = 1024;
 
+{
+  const Sign = crypto.Sign;
+  const instance = Sign('SHA256');
+  assert.ok(instance instanceof Sign, 'Sign is expected to return a new \
+      instance when called without `new` keyword');
```

Comment:
Sorry, two more things:

1. `assert` instead of `assert.ok`
2. We usually try to break strings using + and line them up beneath each other, so something like this:

```js
assert(instance instanceof Sign, 'Sign is expected to return a new instance ' +
                                 'when called without `new` keyword');
```

Same below. Thanks!

Reply:
I don't mind.

> assert instead of assert.ok

What difference between `assert.ok` and `assert` ?
I think `assert()` is better because there are fewer characters, right ?

Reply:
`assert.ok` is just an alias for `assert`. Not better but I think in general we try to use `assert`.

Reply:
Thanks for your detailed information.
I understood that `assert` was better.